mlflow/protos/databricks.proto

```diff
@@ -229,6 +229,9 @@ enum ErrorCode {
   // If the API request is rejected due to throttling
   REQUEST_LIMIT_EXCEEDED = 1007;
 
+  // Indicates that the given API endpoint requires auth credentials
+  AUTHORIZATION_REQUIRED = 1008;
+
```

Comment:
@balvisio Looks like https://github.com/mlflow/mlflow/pull/6106/files#diff-1bf3cdd39aa480c64e304f8e7075e7592f7b7dcc2e2bdf80ae0e242914eb23ecR227 (`CUSTOMER_UNAUTHORIZED = 1006;`) covers this case already. Can we use that error status value instead?

Reply:
@dbczumar Sure. I missed it. I will revert the change here.